.github/CODEOWNERS

```
@@ -1 +1,3 @@
 * @mozilla/fxa-devs
+
+*.ftl @flodolo @bcolsson
```

```suggestion
*.ftl @mozilla/fx-l10n
```

At this point, let's use a group here, since it helps with future maintenance. I don't think that impacts the decision of how to add write rights (via individuals added to fxa-write, or giving write rights to this group).